integration/ui/satellite/e2e_onboarding_test.go

```diff
@@ -0,0 +1,222 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	assert2 "github.com/stretchr/testify/assert"
+	"github.com/zeebo/assert"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestE2eUserCreateLoginAccessInBrowser(t *testing.T) {
```

Comment:
Try to minimize the test name. It's clear from the package path that this is a browser and e2e test. It would be sufficient to have `TestUserRegistration` as the test name.